Homework7/task1.js

```diff
@@ -0,0 +1,43 @@
+export default class Book {
+    constructor(name,author,dateOfCreating){
+        this._name = name;
+        this._author = author;
+        this._dateOfCreating = dateOfCreating;
+    }
+    static getOidestBook(books) {
+        if (!books.length) {
+            return null;
+        }
+        const sorterBooks = [...books].sort((a,b)=> a.dateOfCreating - b.dateOfCreating);
+        return sorterBooks[0];
+    }
+     get name(){
+         return this._name;
+     }
+         set name(value) {
+	 	if (typeof value !== "string" ) {
+		console.log('The books name should be the text');
+		return;
+		}
+        this._name = value;
+    }
+    set author(value1) {
```

Comment:
why you have named it `value1`?
Use more informative name like `name`, `authorName` or just `value` without numbers